dev/NavigationView/NavigationView.cpp

```diff
@@ -1358,6 +1362,36 @@ void NavigationView::UpdateIsClosedCompact()
     }
 }
 
+void NavigationView::UpdatePaneButtonsWidths()
+{
+    auto newButtonWidths = [this]()
+    {
+        if (DisplayMode() == winrt::NavigationViewDisplayMode::Minimal)
+        {
+            return (double)c_paneToggleButtonWidth;
```

Comment:
>(double)c_paneToggleButtonWidth [](start = 19, length = 31)

Better would be `static_cast<double>(c_paneToggleButtonWidth)`

Reply:
Fixed.